README.rst

```diff
@@ -181,26 +181,23 @@ If you do not have **hdf5** or **netcdf4** installed, you can use conda to insta
 
 ::
 
-    conda install -c conda-forge hdf5
-    conda install -c conda-forge netcdf4
+    conda install hdf5 netcdf4
```

Comment:
instead of deleted the original instructions, could you just add a section that is specific to oscar?